sources/org.osbuild.ostree

```diff
@@ -42,6 +42,26 @@ SCHEMA = """
                 "type": "string",
                 "description": "content URL of the repository."
               },
+              "tls-permissive": {
+                "type": "boolean",
+                "description": "By default, server TLS certificates will be checked against the system certificate store. If this variable is set, any certificate will be accepted."
+              },
+              "tls-ca": {
+                "type": "string",
+                "description": "Trusted anchors in PEM format to use instead of system CA database when making requests to this repository."
+              },
+              "tls-client-cert": {
```

Comment:
This way the cert/key will end up in the manifest, in these cases it's ok to work via the environment.

Just reuse the `secrets` property, and if it equals `org.osbuild.mtls` look for the revelant env variables. For curl we do ```
            key = os.getenv("OSBUILD_SOURCES_CURL_SSL_CLIENT_KEY")
            cert = os.getenv("OSBUILD_SOURCES_CURL_SSL_CLIENT_CERT")
            if not (key and cert):
                raise RuntimeError(f"mtls secrets required but key ({key}) or cert ({cert}) not defined")
            desc["secrets"] = {
                'ssl_ca_cert': os.getenv("OSBUILD_SOURCES_CURL_SSL_CA_CERT"),
                'ssl_client_cert': cert,
                'ssl_client_key': key,
            }

```